config/lib/i18n.js

```diff
@@ -33,7 +37,7 @@ i18n
   .use(backend)
   .use(reactI18nextModule) // passes i18n down to react-i18next
   .init({
-    lng: 'eng',
+    lng: languages[0].code,
```

Comment:
Maybe just export this code as a constant from `i18n-languages.js` and we could then let languages be sorted alphabetically?

Reply:
d7e958a8c2b75fdd0ae7b775eb5df7c28790ae03